### PR TITLE
Add grunt file

### PR DESCRIPTION
**Pull request by mcrinquand**

### CODE REVIEW

File: Command/AbstractInitializerCommand.php [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    }
+
+    /**
+     * Check if the task runs in the correct environment
+     *
+     * @param  string $environment
+     *
+     * @throws EnvironmentException
+     */
+    protected function checkEnvironment($environment)
```
Comment (mcrinquand) [inline Command/AbstractInitializerCommand.php]: to delete

File: Command/AbstractInitializerCommand.php [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    protected function init(InputInterface $input, OutputInterface $output)
+    {
+        $this->input  = $input;
+        $this->output = $output;
+    }
+
+    /**
+     * @return ConfigurationManager
+     */
+    protected function getConfigurationManager()
```
Comment (mcrinquand) [inline Command/AbstractInitializerCommand.php]: to delete

**Comment by mcrinquand** [comment]
@nicolas-bastien you can check this

**Comment by nicolas-bastien** [comment]
@mcrinquand you should change the documentation here you speak about this bundle doc so it should be Resources/doc/grunt and in english please ;-)

**Comment by mcrinquand** [comment]
You can check

**Pull request merged**